sdl2promela/sdl/model.py

```diff
@@ -738,6 +792,46 @@ def convert(source: ogAST.ExprAssign):
     return __make_binary_expression(source)
 
 
+@dispatch(ogAST.ExprXor)
+def convert(source: ogAST.ExprXor):
+    return __make_binary_expression(source)
+
+
+@dispatch(ogAST.ExprOr)
+def convert(source: ogAST.ExprOr):
+    return __make_binary_expression(source)
+
+
+@dispatch(ogAST.ExprAnd)
+def convert(source: ogAST.ExprAnd):
+    return __make_binary_expression(source)
+
+
+@dispatch(ogAST.ExprImplies)
+def convert(source: ogAST.ExprImplies):
+    expression = BinaryExpression()
```

Comment:
Please explain why this is not handled by __make_binary_expression

Reply:
Added.